eos-core-depends

```diff
@@ -172,6 +172,7 @@ openprinting-ppds
 openssh-server
 # For flatpak-builder
 patch
+podman
```

Comment:
Won't you also want to include `podman-docker` once that's in the distro?

Reply:
Resolution: no, because without `docker-compose` support, having `/usr/bin/docker` provides false hope.